h/templates/accounts/developer.html.jinja2

```diff
@@ -0,0 +1,51 @@
+{% extends "h:templates/layouts/profile.html.jinja2" %}
+
+{% set page_route = 'profile_developer' %}
+{% set page_title = 'Developer' %}
+
+{% block page_content %}
+  <div class="form-vertical">
+    <h2 class="form-heading">
+      <span>{% trans %}Generate your API token{% endtrans %}</span>
+    </h2>
+
+    {% if token %}
+      <label for="token">{% trans %}Your API token is:{% endtrans %}</label>
+      <input id="token" type="text" value="{{ token }}" readonly="readonly"
+             class="developer-page__api-token-input">
+      <span class="help-block">
+        Please keep your API token safe as it can be used to access your
```

Comment:
I think this should be a bit more explicit about what capabilities the API token has. eg. "... can be used to read, modify and delete all data in your account"